src/main/java/net/mcreator/ui/views/ArmorImageMakerView.java

```diff
@@ -61,11 +61,11 @@ public ArmorImageMakerView(final MCreator fra) {
 
 		type1.setOpaque(false);
 
-		controls.add(new JLabel("Armor texture type: "));
+		controls.add(L10N.label("dialog.armor_image_maker.type"));
 		controls.add(str);
-		controls.add(new JLabel("Texture color: "));
+		controls.add(L10N.label("dialog.armor_image_maker.color"));
 		controls.add(PanelUtils.join(col));
-		controls.add(new JLabel("Lock saturation and lightness: "));
+		controls.add(L10N.label("dialog.armor_image_maker.saturation_lightness_lock"));
```

Comment:
You could probably use "dialog.imageeditor.saturation_brightness_lock" key for this

Reply:
Well this file is not ImageEditor but ArmorImageMaker so why?

Reply:
It refers to the identical parameter in the backend and always will

Reply:
Ok that one does not have : in it, it is ok as it is.